### DISCUSSION TITLE
Reducing clips splitting

### DISCUSSION THREAD

**Discussion by maxdd** [blakeblackshear/frigate#10976] [General]
Hello,
in my testing environment i have a lot of "splitted" small clips.
When i look at them they generally appear to be part of the same scene (which can last even 40-50s).
To give you a general idea, i have a garden where people might come in and out but for whathever reason frigate split this detection into many clips.

Which config parameters shall i look into to make sure that i can reduce the number of clips and condense them into a single one that represent the same scene? Or in general how can i debug this?

Here is an example of a full scene splitted into 4 clips

![image](https://github.com/blakeblackshear/frigate/assets/6155973/015174ec-53c0-4340-8f43-477b3fe37b04)

Is it motion related?

**Comment by NickM-27** [comment]
You can try looking at max_disappeared but in general an event ending means the object stopped being detected
